### PR TITLE
[PLT-6898] Fix JS error when saving a custom theme vector

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix JS error when saving a custom theme vector

#### Ticket Link
Jira ticket: [PLT-6898](https://mattermost.atlassian.net/browse/PLT-6898)

### CODE REVIEW

File: webapp/components/user_settings/user_settings_theme.jsx [hunk 1]
```diff
@@ -125,11 +125,6 @@ export default class ThemeSetting extends React.Component {
                 this.originalTheme = Object.assign({}, this.state.theme);
                 this.scrollToTop();
                 this.props.updateSection('');
-            },
-            (err) => {
```
Comment (hmhealey) [inline webapp/components/user_settings/user_settings_theme.jsx]: And here?

File: webapp/actions/user_actions.jsx [hunk 2]
```diff
@@ -396,21 +396,18 @@ export function loadProfilesForDM() {
...
         category: Preferences.CATEGORY_THEME,
         name: teamId,
         value: JSON.stringify(theme)
-    }])(dispatch, getState).then(
-        (data) => {
-            if (data && onSuccess) {
-                onThemeSaved(teamId, theme, onSuccess);
-            } else if (data == null && onError) {
-                const serverError = getState().requests.users.savePreferences.error;
-                onError({id: serverError.server_error_id, ...serverError});
```
Comment (hmhealey) [inline webapp/actions/user_actions.jsx]: Why are we removing the error handling here?

**Comment by hmhealey** [comment]
Approving this as we'll discuss error handling more next week

File: webapp/components/user_settings/user_settings_theme.jsx (hunk 1 above)
Comment (saturninoabril) [inline webapp/components/user_settings/user_settings_theme.jsx]: As I understood, on success and on error are currently not supported in mattermost-redux; however, rollback is available whenever failure on saving occurred.
But I agree that on success and on error are something that should be added.

File: webapp/actions/user_actions.jsx (hunk 2 above)
Comment (saturninoabril) [inline webapp/actions/user_actions.jsx]: same as above.

**Pull request merged**